legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -278,7 +278,7 @@ def get_office_revision(filing, business) -> dict:  # pylint: disable=too-many-l
 
         offices_current = (
             db.session.query(Office, null().label("changed"))
-            .filter(Office.change_filing_id == filing_id)
+            .filter(Office.change_filing_id <= filing_id)
```

Comment:
I am not sure about this. @argush3

Reply:
@vysakh-menon-aot let me know what issues you see with the updates Ketaki made when you have a minute.

I have attached an SP example below and although it is not directly applicable to the filings being tested, I think it still illustrates the retrieval of current and versioning data for offices.

Assuming the filing of interest was the most recent filing, I believe the updates Ketaki made here would be able to pick up the correct current and historical office entries.

![image](https://github.com/bcgov/lear/assets/6685223/353fd98d-232a-400e-9c12-18cd141bb47c)

Reply:
looks good